db/schema.rb

```diff
@@ -240,8 +240,8 @@
     t.boolean  "is_prediction",            :default => false
```

Comment:
This schema change was checked in 'by accident' -- but was never edited by hand, and probably reflects that someone else didn't check it in earlier?